src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class Header extends React.Component {
```

Comment:
Awesome to separate this out into its own component. If you wanted, you could make this a simple functional component like:
`const Header = () =>  <h1>Eggheadded</h1>`